nixos/modules/security/wrappers/default.nix

```diff
@@ -41,7 +39,7 @@ let
       };
     options.permissions = lib.mkOption
       { type = fileModeType;
-        default  = "u+rx,g+x,o+x";
+        default  = "u+rx,g+rx,o+rx";
```

Comment:
I think it might be smarter to ensure that it will always stay readable, rather than just changing it here...